crates/ruff/src/rules/pycodestyle/snapshots/ruff__rules__pycodestyle__tests__tab_size_8.snap

```diff
@@ -0,0 +1,72 @@
+  |
+
+E501_2.py:2:77: E501 Line too long (101 > 88 characters)
+  |
+2 | a = 5678901234567890123456789012345678901234567890123456789012345678901234567890123456789
+3 | a   =   5678901234567890123456789012345678901234567890123456789012345678901234567890123456789
+  |                                                                                 ^^^^^^^^^^^^^ E501
+4 | 
+5 | b = 5678901234567890123456789012345678901234567890123456789012345678901234567890123456789
+  |
+
+E501_2.py:4:89: E501 Line too long (89 > 88 characters)
+  |
+4 | a   =   5678901234567890123456789012345678901234567890123456789012345678901234567890123456789
+5 | 
+6 | b = 5678901234567890123456789012345678901234567890123456789012345678901234567890123456789
+  |                                                                                         ^ E501
+7 | b   =   5678901234567890123456789012345678901234567890123456789012345678901234567890123456789
+  |
+
+E501_2.py:5:77: E501 Line too long (101 > 88 characters)
+  |
+5 | b = 5678901234567890123456789012345678901234567890123456789012345678901234567890123456789
+6 | b   =   5678901234567890123456789012345678901234567890123456789012345678901234567890123456789
+  |                                                                                 ^^^^^^^^^^^^^ E501
+7 | 
+8 | c   =   901234567890123456789012345678901234567890123456789012345678901234567890123456789
+  |
+
+E501_2.py:7:89: E501 Line too long (89 > 88 characters)
+   |
+ 7 | b   =   5678901234567890123456789012345678901234567890123456789012345678901234567890123456789
+ 8 | 
+ 9 | c   =   901234567890123456789012345678901234567890123456789012345678901234567890123456789
+   |                                                                                         ^ E501
+10 | c   =   901234567890123456789012345678901234567890123456789012345678901234567890123456789
+   |
+
+E501_2.py:8:77: E501 Line too long (97 > 88 characters)
+   |
+ 8 | c   =   901234567890123456789012345678901234567890123456789012345678901234567890123456789
+ 9 | c   =   901234567890123456789012345678901234567890123456789012345678901234567890123456789
+   |                                                                                 ^^^^^^^^^ E501
+10 | 
+11 | d       =       7890123456789012345678901234567890123456789012345678901234567890123456789
+   |
+
+E501_2.py:10:89: E501 Line too long (89 > 88 characters)
+   |
+10 | c   =   901234567890123456789012345678901234567890123456789012345678901234567890123456789
+11 | 
+12 | d       =       7890123456789012345678901234567890123456789012345678901234567890123456789
+   |                                                                                         ^ E501
+13 | d   =   7890123456789012345678901234567890123456789012345678901234567890123456789
+   |
+
+E501_2.py:11:77: E501 Line too long (89 > 88 characters)
+   |
+11 | d       =       7890123456789012345678901234567890123456789012345678901234567890123456789
```

Comment:
Can we throw in some unicode, just to makes sure we handle it correctly

Reply:
Done!
The ranges in the snapshots are ok but not in VSCode.
![Screenshot from 2023-05-20 17-11-43](https://github.com/charliermarsh/ruff/assets/13716151/f5180443-f469-41e1-a3d1-1fda87467b08)

Reply:
VSCode only counts one character for 💣